From 47a2dbb02c71bd5be1bff7b295a03526dcbe5507 Mon Sep 17 00:00:00 2001
From: Marcus Lundblad <ml@update.uu.se>
Date: Wed, 22 Jun 2016 23:25:08 +0200
Subject: [PATCH] mapView: Stop using MapQuest tiles

Revert to using the standard OSM Mapnik tiles instead of MapQuest's
tiles, as MapQuest will cease providing free tiles.

https://bugzilla.gnome.org/show_bug.cgi?id=764841
---
 src/mapView.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/mapView.js b/src/mapView.js
index aa587da..09af348 100644
--- src/mapView.js
+++ src/mapView.js
@@ -47,7 +47,7 @@ const Utils = imports.utils;
 
 const MapType = {
     LOCAL: 'MapsLocalSource',
-    STREET:  Champlain.MAP_SOURCE_OSM_MAPQUEST,
+    STREET:  Champlain.MAP_SOURCE_OSM_MAPNIK,
     AERIAL:  Champlain.MAP_SOURCE_OSM_AERIAL_MAP,
     CYCLING: Champlain.MAP_SOURCE_OSM_CYCLE_MAP,
     TRANSIT: Champlain.MAP_SOURCE_OSM_TRANSPORT_MAP
-- 
2.8.1
